lib/omnibus/semantic_version.rb

```diff
@@ -0,0 +1,26 @@
+module Omnibus
+  class SemanticVersion
+    def initialize(version_string)
+      @has_v = !!(version_string =~ /^v/)
+      @version = Gem::Version.new(version_string.gsub(/^v/, ""))
```

Comment:
Pretty sure this will choke on SemVer-compliant pre-release and build versions. This would probably be a good time to begin using [mixlib-versioning](https://github.com/chef/mixlib-versioning) for Omnibus's internal version parsing.